ci/qa/create-db

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
+
+cd $(dirname $0)/../../
+
+bin/console doctrine:schema:drop --env=test --force
+bin/console doctrine:schema:create --env=test
```

Comment:
I love this scritpt! It might be a good idea to emphasize the fact this is only used for test environments. So maybe change filename to: `create-test-db`. If you'd like to support creating `test` and  `dev` databases, then consider allow setting of the environment via a command argument. But please whitelist any allowed environment in the script. You don't want to run the risk resetting a prod database with this script.

Reply:
For now i will refactor the name. Though i will consider adding this in the future.